particles/Tutorial/Javascript/4_Handles/source/GetPerson.js

```diff
@@ -10,25 +10,32 @@
 
  /* global defineParticle */
 
- defineParticle(({SimpleParticle, html}) => {
 
-    const template = html`
-  <input value="{{name}}" placeholder="Enter your name" spellcheck="false" on-change="onNameInputChange">
-  <div slotid="greetingSlot"></div>
-    `;
+defineParticle(({SimpleParticle, html}) => {
+
+  const template = html`
+<input placeholder="Enter your name" spellcheck="false" on-change="onNameInputChange">
+<div slotid="greetingSlot"></div>
+  `;
 
   return class extends SimpleParticle {
     get template() {
       return template;
     }
 
-    update(props, state) {
-      // Get the handle "person" and update the data stored to be the name of the person we will greet.
-      this.set('person', {name: 'Human'});
+    // Because we purely have some logic to implement, we use update instead of render. 
```

Comment:
What does 'purely' sound strange to me here, though I am a non native speaker so that's maybe why. WDYT?